src/module/ovs/event.rs

```diff
@@ -107,6 +134,18 @@ pub(crate) struct UpcallEvent {
 }
 unsafe impl Plain for UpcallEvent {}
 
+impl EventFmt for UpcallEvent {
+    fn event_fmt(&self, f: &mut fmt::Formatter, _: DisplayFormat) -> fmt::Result {
+        write!(
+            f,
+            "UPCALL{} port {} cpu {}",
```

Comment:
The rest of the events are using lowercase. OVS should be kept that way as well.